cli/dcoscli/package/main.py

```diff
@@ -510,6 +510,13 @@ def _uninstall(package_name, remove_all, app_id, cli, app, skip_confirmation):
         return 0
 
     package_manager = get_package_manager()
+    # When no --cli or --app flag is passed, we set app to True and
+    # cli to None in order to allow the uninstall function to make
+    # the best choice.
+    if app is False and cli is False:
+        cli = None
+        app = True
+
```

Comment:
I don't think we should do this logic out here, I would rather push this down into the actual `package.uninstall()` command.